storage/innobase/handler/ha_innodb.cc

```diff
@@ -3639,6 +3639,16 @@ static uint innobase_partition_flags()
 	return (0);
 }
 
+/** Return the minimum buffer pool size based on page size */
+static inline ulint min_buffer_pool_size()
+{
+	ulint s= (BUF_LRU_MIN_LEN + BUF_LRU_MIN_LEN / 4) * srv_page_size;
+	/* buf_pool_chunk_size mimimum is 1M, so round up to a multiple */
```

Comment:
Typo: "minimum", not "minmimum". In the commit message it is mistyped in a different way.

Furthermore, TAB should not be used for indenting new code. Please use 2 spaces instead.